graylog2-server/src/main/java/org/graylog/events/notifications/types/EmailSender.java

```diff
@@ -180,9 +180,10 @@ void sendEmails(EmailEventNotificationConfig notificationConfig, EventNotificati
 
         if (emailRecipients.isEmpty()) {
             if (!emailConfig.isEnabled()) {
```

Comment:
why is this only checked if the recipients are empty? looks like a bug.

Reply:
It's the behaviour of the current EmailAlarmCallback, but I guess we can change it. Sounds also better to me to check it independently.  https://github.com/Graylog2/graylog2-server/blob/master/graylog2-server/src/main/java/org/graylog2/alarmcallbacks/EmailAlarmCallback.java#L91